script.skin.helper.service/default.py

```diff
@@ -99,36 +114,50 @@ def __init__(self):
                 if resultAction:
                     if "jsonrpc" in resultAction:
                         xbmc.executeJSONRPC(resultAction)
+                        xbmc.executeJSONRPC(resultAction)
```

Comment:
Oh well, I just remember..it does start but stops again after 1 second, weird huh...
Anyway I removed the extra line for now. We'll just wait untill someone actually complains about it.